### PR TITLE
Inline Teacher URI Option; CLI Touch-ups

### PR DESCRIPTION
**Pull request by KPrasch**
##### Touch Ups
* Deprecate CLI Simulate Command
* Accept Load-Balanced Teacher URI's 
* Cleanup Simple Observer Output

##### Temporary Fixes
* Skip S3 Storage Tests (Recent Version Incompatibility with `boto`, `botocore`, `moto`) #527
* Include 'cli' as a top-level module.  #502 
* Pin `tzlocal==1.5.1` (Version Conflicts unresolved via Pipenv)

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ pysha3="*"
 requests = "*"
 sqlalchemy = "*"
 apistar = "==0.5.42"
-tzlocal = "==2.0.0b1"
+tzlocal = "==1.5.1"
```
Comment (michwill) [inline Pipfile]: Later I'd certainly try `"<2.0.0"`. But it works for now

Comment (KPrasch) [reply Pipfile]: :+1: Agreed

File: cli/main.py [hunk 2]
```diff
@@ -678,217 +681,6 @@ def stake(config,
...
-            
-            Simulation processes .............. {}
-            
-            """.format(ursula_processes)
-        click.echo(status_message)
-
-    else:
-        raise click.BadArgumentUsage
-
-
```
Comment (jMyles) [inline cli/main.py]: Phew!  That's a breath of fresh air right there.  A goddam strawberry patch of redlines in here.

File: cli/main.py [hunk 3]
```diff
@@ -933,14 +933,12 @@ def status(config,
 
 @cli.command()
 @click.option('--debug', is_flag=True)
-@click.option('--trusted-teacher-uri', type=click.STRING)
+@click.option('--teacher-uri', type=click.STRING)
+@click.option('--min-stake', type=click.INT, default=0)
```
Comment (jMyles) [inline cli/main.py]: It's weird and wild and I think I like it.

File: cli/main.py [hunk 4]
```diff
@@ -1018,14 +1009,31 @@ def __make_ursula():
...
-        seed_metadata = [SeednodeMetadata(address, host, port)]  # TODO: Ensure seed injection works
-    else:
-        seed_metadata = ()
+    #
+    # Seed
+    #
+    seed_metadata, teacher_nodes = list(), list()
+    if teacher_uri:
+
+        if all(('@' in teacher_uri, ':' in teacher_uri)):
```
Comment (jMyles) [inline cli/main.py]: This seems like odd logic to me.  If they don't specify a port (but do specify a checksum address) then we shove them into `from_seed_and_stake_info`?

File: Pipfile (hunk 1 above)
Comment (jMyles) [inline Pipfile]:
```suggestion
tzlocal = "<2.0.0"
```

I mean... I'll make it.  :-)

File: cli/main.py (hunk 4 above)
Comment (michwill) [reply cli/main.py]: Validation regex?

File: Pipfile (hunk 1 above)
Comment (michwill) [reply Pipfile]: Also in `setup.py`. And, if it doesn't break stuff (@KPrasch was saying that it does)

File: cli/main.py (hunk 4 above)
Comment (KPrasch) [reply cli/main.py]: Yeah - this needs a bit of work, fixing #522 will start to open this logic a bit

File: Pipfile (hunk 1 above)
Comment (KPrasch) [reply Pipfile]: Leaving this until later... worried about keeping a good set a dependencies pre-cedar

**Pull request merged**